src/asciidoc/docs/blueprints/customVCC.adoc

```diff
@@ -0,0 +1,208 @@
+= Custom VCC Template
+:page-slug: custom-vcc-blueprint
+
+Follow this blueprint to build a template for a custom Koji VCC to give remixes the option to select from a drop down of choices which are retrieved from a third party API.
+
+image::vote-counter.png[Make a custom VCC on Koji,role="img-overview"]
+
+=== Prerequisites
+
+* Familiarity with web development.
+React and ES6 basics are a plus.
+* Understanding of third party API usage beneficial.
+* Familiarity with the Koji remix process.
+For an overview, see the <<start-guide-1#, starter guide>>.
+* Familiarity with Koji custom VCCs is a bonus.
+For an overview, see <<build-custom-vcc#>>.
+
+=== Level
+
+* *Koji*: Intermediate – Advanced
+* *Developer*: Intermediate – Advanced
+* *Time*: 30 minutes
+
+== Building blocks
+
+=== Remix the scaffold
+
+Remix an existing web application on Koji that implements basic elements of your favorite framework.
+
+.React
+https://withkoji.com/templates/seane/simple-react-scaffold-remix-[Simple React scaffold]
+
+.Vanilla JS
+https://withkoji.com/templates/JamesHole/vanilla-js-scaffold-with-vccs[Vanilla JS scaffold]
+
+=== Install the packages
+
+Install `@withkoji/custom-vcc-sdk` to enable custom VCC functionality such as loading and saving the VCC value from Koji and install the development dependency @babel/plugin-proposal-class-properties to add support for some extra syntax to be compiled by babel.
+
+[source,bash]
+----
+npm install --save @withkoji/custom-vcc-sdk
+npm install --save-dev @babel/plugin-proposal-class-properties
+----
+
+=== Update the runtime configuration
+Add the babel plugin to the babel runtime configuration file to ensure it can use it when compiling
+
+[source,json]
+----
+include::../_includes/vcc-blueprint-babelrc.json[]
+----
+
+=== Create the customization files
+Create `.koji/customization/settings.json` to define customizable values for the API URL, and the items key and item name key from the API response.
+
+[source,json]
+----
+include::../_includes/vcc-blueprint-settings.json[tags=!configuration]
+----
+<1> Customizations will be added here in the next step
+
+Remove the unused customization files.
+
+`colors.json`, `images.json`, and `strings.json`
+
+=== Configure the API
+For this blueprint we implement https://thecatapi.com/[The Cat API] which can return a list of cat breeds but you can use any API that returns a list of items.
```

Comment:
As a style convention, I've decided to use "you" instead of "we" for the narrative instructions.

```suggestion
This blueprint implements https://thecatapi.com/[The Cat API] to return a list of cat breeds.
However, you can use any API that returns a list of items to implement a custom selector.
```